### ISSUE TITLE
Issue loading fixture, while calling notify_pending_status_update()

### ISSUE THREAD

**Issue by tristanlatr** [LesFruitsDefendus/saskatoon-ng#247]
From what I understand, looks like this function is being called when loading the fixture generating an error because the object is not in the database at this moment. 

See the traceback here: https://github.com/LesFruitsDefendus/saskatoon-ng/discussions/240#discussioncomment-2568292

@victorphoenix3, would you be OK looking at this since you are familiar with this code?

Thanks,

**Comment by tristanlatr** [comment]
I've been able to reproduce this issue inside the github action runner:

```
Run saskatoon/manage.py loaddata saskatoon/fixtures/saskatoon.json
Traceback (most recent call last):
  File "saskatoon/manage.py", line 21, in <module>
    main()
  File "saskatoon/manage.py", line 1[7](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:7), in main
    execute_from_command_line(sys.argv)
  File "/opt/hostedtoolcache/Python/3.[8](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:8).12/x64/lib/python3.8/site-packages/django/core/management/__init__.py", line 41[9](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:9), in execute_from_command_line
    utility.execute()
  File "/opt/hostedtoolcache/Python/3.8.[12](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:12)/x64/lib/python3.8/site-packages/django/core/management/__init__.py", line 4[13](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:13), in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/core/management/commands/loaddata.py", line 78, in handle
    self.loaddata(fixture_labels)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/core/management/commands/loaddata.py", line 123, in loaddata
    self.load_label(fixture_label)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/core/management/commands/loaddata.py", line 190, in load_label
    obj.save(using=self.using)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/core/serializers/base.py", line 223, in save
    models.Model.save_base(self.object, using=using, raw=True, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/db/models/base.py", line 763, in save_base
    pre_save.send(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/dispatch/dispatcher.py", line [18](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:18)0, in send
    return [
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/dispatch/dispatcher.py", line 181, in <listcomp>
    (receiver, receiver(signal=self, sender=sender, **named))
  File "/home/runner/work/saskatoon-ng/saskatoon-ng/saskatoon/harvest/signals.py", line 44, in notify_pending_status_update
    original_instance = sender.objects.get(id=instance.id)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/django/db/models/query.py", line 4[35](https://github.com/LesFruitsDefendus/saskatoon-ng/runs/6028737427?check_suite_focus=true#step:9:35), in get
    raise self.model.DoesNotExist(
harvest.models.DoesNotExist: Problem installing fixture '/home/runner/work/saskatoon-ng/saskatoon-ng/saskatoon/fixtures/saskatoon.json': Property matching query does not exist.
````

**Comment by tristanlatr** [comment]
see #250

**Comment by 2ynn** [comment]
@tristanlatr I think it must have to do with the fact I merged victorphoenix3's branch with `notify_status_pending` into develop instead of the feature branch I'm working on. Let me merge  #234 first, it should fix the issue

**Comment by 2ynn** [comment]
concerning loading initial fixtures I suggest you run the `saskatoon/fixtures/init` script instead.  I don't think we should keep maintaining `saskatoon.json`.

**Comment by 2ynn** [comment]
By the way on the develop branch it doesn't seem possible to start from a fresh database anymore.
I think It has to do with:
```
    seasons = []
    for y in Harvest.objects.all():
       [...]
```

**Comment by 2ynn** [comment]
fixed in #234
